samples/browser/src/main/java/org/mozilla/samples/browser/BaseBrowserFragment.kt

```diff
@@ -207,6 +209,18 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler {
             view = layout
         )
 
+        appLinksFeature.set(
```

Comment:
Is this needed? I don't see any usage of this.

Reply:
@rocketsroger See my earlier comment: https://github.com/mozilla-mobile/android-components/pull/7367#discussion_r442032905

Reply:
Sorry, I didn't see that comment.  My mistake.  Please take a look at:
https://github.com/mozilla-mobile/android-components/blob/552b63f50640604bef348e56ebe7cb2eb6fd015c/components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksInterceptor.kt#L86-L87

This might fit what you want.  I'm not sure if you want your feature to be tied to app links.

Reply:
@rocketsroger thanks for the input. I refactored this to take a `launchFromInterceptor` like the `AppLinksInterceptor`.

https://github.com/mozilla-mobile/android-components/pull/7367/files#diff-ff99e8e29f17903954c251d6b5aa67e3R33-R46

This way this feature does not depend on the `AppLinksFeature`, but if desired the intent could be launched via the app links feature. I think this is important, because the app links feature already has some behavior which might also be desired for Web Apps:
https://github.com/mozilla-mobile/android-components/blob/29cc847c254a78685f5a3bb80b4c21e47c52deb9/components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksFeature.kt#L46-L75

Reply:
Looks good to me.  Thanks